packages/sdk/utils/tsconfig.json

```diff
@@ -2,7 +2,8 @@
   "extends": "@celo/typescript/tsconfig.library.json",
   "compilerOptions": {
     "rootDir": "src",
-    "outDir": "lib"
+    "outDir": "lib",
+    "downlevelIteration": true
```

Comment:
Is this for handling accented chars better? I wish json allowed comments

Reply:
It's to allow for iterating over generator functions